multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/SpaceWithUser.java

```diff
@@ -0,0 +1,40 @@
+package org.cloudfoundry.multiapps.controller.web.security;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public class SpaceWithUser {
+
+    private final UUID userGuid;
+    private final UUID spaceGuid;
+
+    public SpaceWithUser(UUID userGuid, UUID spaceGuid) {
+        this.userGuid = userGuid;
+        this.spaceGuid = spaceGuid;
+    }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (object == null || getClass() != object.getClass()) {
+            return false;
+        }
+        SpaceWithUser spaceWithUser = (SpaceWithUser) object;
+        return Objects.equals(userGuid, spaceWithUser.userGuid) && Objects.equals(spaceGuid, spaceWithUser.spaceGuid);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(userGuid, spaceGuid);
+    }
+
+    public UUID getUserGuid() {
```

Comment:
I think the convention is to have any getters / other methods before `equals` and `hashCode`

Reply:
Done